apps/frontend/src/components/global/ExportHTMLModal.vue

```diff
@@ -0,0 +1,327 @@
+    const statusCardHTML = document.getElementById('statusCounts')?.innerHTML
+    const statusCountCardContainer = exportWindow.document.getElementById('statusCountCard')
+    const severityCardHTML = document.getElementById('severityCounts')?.innerHTML
+    const severityCardContainer = exportWindow.document.getElementById('severityCard')
+    const complianceCardHTML = document.getElementById('complianceLevel')?.innerHTML
+    const complianceCardcontainer = exportWindow.document.getElementById('complianceCard')
+    if(statusCardHTML && severityCardHTML && complianceCardHTML) {
+      // Append the HTML to each container
+      statusCountCardContainer?.insertAdjacentHTML('afterbegin', statusCardHTML)
+      severityCardContainer?.insertAdjacentHTML('afterbegin', severityCardHTML)
+      complianceCardcontainer?.insertAdjacentHTML('afterbegin', complianceCardHTML)
+      // There seems to be some sort of automatic formatting applied with .resize-triggers that doesn't apply here
+      const elements = exportWindow.document.getElementsByClassName('resize-triggers');
+      while (elements[0]) {
+        const element = elements[0];
+        element?.parentNode?.removeChild(element);
+      }
+    }
+  }
+
+    addDetails(control: ContextualizedControl): ContextualizedControl & {details: Detail[]} {
+      return {...control, full_code: control.full_code, details: [
+      {
+        name: 'Control',
+        value: control.data.id
+      },
+      {
+        name: 'Title',
+        value: control.data.title
+      },
+      {
+        name: 'Caveat',
+        value: control.hdf.descriptions.caveat
+      },
+      {
+        name: 'Desc',
+        value: control.data.desc
+      },
+      {
+        name: 'Rationale',
+        value: control.hdf.descriptions.rationale
+      },
+      {
+        name: 'Justification',
+        value: control.hdf.descriptions.justification
+      },
+      {
+        name: 'Severity',
+        value: control.root.hdf.severity
+      },
+      {
+        name: 'Impact',
+        value: control.data.impact
+      },
+      {
+        name: 'Nist controls',
+        value: control.hdf.raw_nist_tags.join(', ')
+      },
+      {
+        name: 'Check Text',
+        value: control.hdf.descriptions.check || control.data.tags.check
+      },
+      {
+        name: 'Fix Text',
+        value: control.hdf.descriptions.fix || control.data.tags.fix
+      }
+    ].filter((v) => v.value)}
+  }
+
+  exportHTML(): void {
+    this.resetOutputData();
+    if(this.filter.fromFile.length === 0){
+      return SnackbarModule.failure('No files have been loaded.')
+    }
+    const templateRequest = axios.get(`/static/export/template.html`)
+    const bootstrapCSSRequest = axios.get(`/static/export/style.css`)
+    const bootstrapJSRequest = axios.get(`/static/export/bootstrap.js`)
+    const jqueryRequest = axios.get(`/static/export/jquery.js`)
```

Comment:
Can these be all labeled with the same names that they have from the packages so it is clear what they actually are (including the min part) so `style.css should be `bootstrap.min.css`. Also these files should be `.gitignore`'d so they don't accidentally get committed to the repository.